.docker/Dockerfile.ckan

```diff
@@ -15,13 +16,14 @@ RUN apk add --no-cache curl build-base \
     && rm dockerize-alpine-linux-amd64-${DOCKERIZE_VERSION}.tar.gz
 
 # Install CKAN.
-ENV CKAN_VERSION 2.8.8
 
 RUN . ${VENV_DIR}/bin/activate \
     && pip install setuptools==36.1 \
-    && pip install -e "git+https://github.com/ckan/ckan.git@ckan-${CKAN_VERSION}#egg=ckan" \
+    && pip install -e "git+https://github.com/qld-gov-au/ckan.git@ckan-${CKAN_VERSION}#egg=ckan" \
```

Comment:
Because it needs patching, see https://github.com/ckan/ckan/compare/2.9...qld-gov-au:qgov-master-2.9.4?expand=1

Fixing a race condition when first creating an instance, and handling both Python 2 and 3 correctly.